typescript/cli/src/context/context.ts

```diff
@@ -57,13 +62,18 @@ export async function getContext({
   }
   const multiProvider = await getMultiProvider(registry, signer);
 
+  let submissionStrategy: SubmissionStrategy | undefined = undefined;
+  if (submissionStrategyFilepath)
+    submissionStrategy = getSubmissionStrategy(submissionStrategyFilepath);
```

Comment:
@yorhodes and I chatted about this, and we do expect this will be as widespread as something like `key`! While the transactions.json logic is certainly `submit` command specific, we do expect templated submission strategies will be used throughout the CLI context re CRUD integration